ecs-cli/modules/utils/utils.go

```diff
@@ -19,6 +19,11 @@ import (
 	"os"
 )
 
+const (
+	// EcsCliResourcePrefix is prepended to the names of resources created through the ecs-cli
+	EcsCliResourcePrefix = "amazon-ecs-cli-setup-"
```

Comment:
My eyes don't like this... this should be `ECSCLIResourcePrefix` to fit Golang style for initialisms.